pkg/activator/handler/handler.go

```diff
@@ -61,7 +73,53 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	err := a.Throttler.Try(revID, func() {
-		attempts, httpStatus := a.proxyRequest(w, r, target)
+		var (
+			attempts   int
+			httpStatus int
+		)
+		// If a GET probe interval has been configured, then probe
+		// the queue-proxy with our network probe header until it
+		// returns a 200 status code.
+		success := (a.GetProbeCount == 0)
+		if !success {
+			probeReq := &http.Request{
+				Method:     http.MethodGet,
+				URL:        target,
+				Proto:      r.Proto,
+				ProtoMajor: r.ProtoMajor,
+				ProtoMinor: r.ProtoMinor,
+				Host:       r.Host,
+				Header: map[string][]string{
+					http.CanonicalHeaderKey(network.ProbeHeaderName): []string{"true"},
+				},
+			}
+			settings := wait.Backoff{
```

Comment:
I think this invalidates the comment above where `maxRetries` says that the sum will evaluate to 1 minute. My math shows you'll be getting 37.151 seconds (or 48397, depending on how wait internals are done), but still not a minute.

Reply:
That comment was written for the other code path.  We can clean that up when we remove the other path?